### DISCUSSION TITLE
HTB Designs Not Editable

### DISCUSSION THREAD

**Discussion by p1ercedev** [Syslifters/sysreptor#278] [General]
When editing designs for HTB reports, when editing report fields, the PDF updates no matter how many times I save and refresh pdf

#### Importing Designs
`curl -s "$url" | docker compose exec --no-TTY app python3 manage.py importdemodata --type=design`

![image](https://github.com/Syslifters/sysreptor/assets/82008705/e90c2b20-388f-4bd6-8df1-cda7b4ff651f)

#### Editing title report field
![image](https://github.com/Syslifters/sysreptor/assets/82008705/69c45f88-32f4-4a2b-a4ff-1f8bc6c505d8)

#### Not updated

![image](https://github.com/Syslifters/sysreptor/assets/82008705/cb53ac01-49ad-43c3-83b7-d5b20b6fe94c)

**Thanks everyone for your help!**

**Comment by aronmolnar** [comment]
I guess you're expecting "NEW_TITLE" to pop up at your report.

This is not the case because, that's just the default value of that report field. It would be prefilled if you create a new project.
If you want the new title to appear in your rendered report, you can update the data in the "PREVIEW DATA" tab.

![image](https://github.com/Syslifters/sysreptor/assets/25219407/27d00113-f628-4c29-88ac-83af2aec4f74)

**Reply by p1ercedev** [reply]
> I guess you're expecting "NEW_TITLE" to pop up at your report.
> 
> This is not the case because, that's just the default value of that report field. It would be prefilled if you create a new project. If you want the new title to appear in your rendered report, you can update the data in the "PREVIEW DATA" tab.
> 
> ![image](https://private-user-images.githubusercontent.com/25219407/344660856-27d00113-f628-4c29-88ac-83af2aec4f74.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk4NDU5MDMsIm5iZiI6MTcxOTg0NTYwMywicGF0aCI6Ii8yNTIxOTQwNy8zNDQ2NjA4NTYtMjdkMDAxMTMtZjYyOC00YzI5LTg4YWMtODNhZjJhZWM0Zjc0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzAxVDE0NTMyM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkxZjdiNTQyYzNjODM1YTI5MjdkN2QyMTExZGMzMjAxMWE2MWQ0ODU4NjkyYTUyMjA5NDIyN2YzMzg1ZjdiZDYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.giQ9ch6tEjyfisxscqZnm98p7N1JkJZfFGIWxw-kIck)

🤦  Thank you, sorry for the trouble

**Reply by aronmolnar** [reply]
No worries :)
